### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by nooomski**
Some of the errors in the [Check Markdown Links](https://github.com/cosmos/gaia/actions/runs/1815731321) action are fixed in this PR. Some remain unfixed and need to be ignored.

@mmulji-ic as far as I can tell you haven't yet changed the config to retry on 429s, right? If you're still planning to do that, I would recommend adding the following folders to the `ignore` list in the [config file](https://github.com/tcort/markdown-link-check#config-file-format):

```
docs/migration
docs/governance/proposals
docs/governance/param-changes
docs/zh
docs/es
docs/ko
```

These are historical folders not in use anymore, or translations that are outdated and currently not displayed on the website. Additionally, the following two individual files can also be ignored:

```
docs/roadmap/README.md
docs/governance/state-of-cosmos-governance-2021.md
```

FYI, I could not reproduce the 403 errors in `docs/resources/ledger.md`, so I'm not entirely sure why the check fails here.

`docs/hub-tutorials/deploy-testnet.html` is too big to fix, and needs a complete rewrite. There is a new issue for that [here](https://github.com/cosmos/gaia/issues/1244). In the interim I would suggest also placing this one on the ignore list.

@mmulji-ic feel free to add your config file to my branch so we can merge this in one PR.

### CODE REVIEW

File: docs/launch/blog-2-en.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Once transfers are enabled, state rollbacks will be much more difficult to justi
 
 You can, however, safely delegate Atoms to validators in this phase by following the CLI guideline and video tutorial linked below. Of course, in the event of a state reversion, any earned fees and inflation may be lost. Note that only `gaiad` should be used for making transactions. Voyager, the GUI for interacting with the Cosmos Hub, is currently in alpha and undergoing development. A separate announcement will be made once Voyager is safer for use.
 
-CLI Guide 🔗:  [github.com/cosmos/cosmos-sdk/…/delegator-guide-cli.md](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/delegator-guide-cli.md) 
+CLI Guide 🔗: https://hub.cosmos.network/main/delegators/delegator-guide-cli.html
```
Comment (glnro) [inline docs/launch/blog-2-en.md]: Does it make sense to make this a relative link? `../delegators/delegator-guide-cli.md`

Comment (nooomski) [reply docs/launch/blog-2-en.md]: Yuuup!

**Review by glnro — APPROVED** [review]
Awesome 👍

**Comment by okwme** [comment]
hell yea @nooomski !

**Pull request merged**